prov/efa/src/efa_hmem.c

```diff
@@ -238,6 +238,27 @@ static inline void efa_hmem_info_check_p2p_support_neuron(struct efa_hmem_info *
 	return;
 }
 
+static inline void
+efa_hmem_info_popuplate_iface_string(enum fi_hmem_iface iface, struct efa_hmem_info *info) {
```

Comment:
there is already an `fi_tostr` for hmem interface: `fi_tostr(&iface, FI_TYPE_HMEM_IFACE)`